cmd/fleet/serve.go

```diff
@@ -291,6 +292,13 @@ the way that the Fleet server works.
 			level.Info(logger).Log("component", "redis", "mode", redisPool.Mode())
 
 			ds = cached_mysql.New(ds)
+			var dsOpts []mysqlredis.Option
+			// TODO(mna): clarify condition to add the host count check for hard enrollment limit.
```

Comment:
@chiiph Unclear to me how to enable the enforcement of the hosts limit, how do I detect that this is running for Fleet Demo?

Reply:
We can have a new option/env var `enforce_license_limits` the defaults to false.

Reply:
Not without some help from a variable/param being set.